ui/src/app/pages/artifactVersion/artifactVersion.tsx

```diff
@@ -422,13 +422,15 @@ export class ArtifactVersionPage extends PageComponent<ArtifactVersionPageProps,
                 const artifactVersionLocation: string = `/artifacts/${ encodeURIComponent(groupId) }/${ encodeURIComponent(versionMetaData.id) }/versions/${versionMetaData.version}`;
                 Services.getLoggerService().info("[ArtifactVersionPage] Artifact version successfully uploaded.  Redirecting to details: ", artifactVersionLocation);
                 this.navigateTo(this.linkTo(artifactVersionLocation))();
+                this.setMultiState({uploadFormData: null, isUploadFormValid: false});
```

Comment:
I guess this is needed because the navigation is to the same page and therefore the page state isn't reset?

Reply:
yes, this is fixing a bug